### PR TITLE
fix(cta-card): updating `video-cta-composite` to match new API

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Related Ticket(s)

Web component: Card group - Video links are broken and do not open on lightbox media view. #6674

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/web-components/src/components/cta/video-cta-composite.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ class DDSVideoCTAComposite extends ModalRenderMixin(HostListenerMixin(LitElement
   private async _handleRequestVideoData(event: CustomEvent) {
     const { href } = event.detail;
     (event.target as VideoCTAMixinImpl).videoThumbnailUrl = VideoPlayerAPI.getThumbnailUrl({
```
Comment (annawen1) [inline packages/web-components/src/components/cta/video-cta-composite.ts:80]: Hey @guilhermelMoraes, I'm thinking we probably want to switch out `VideoPlayerAPI` with `KalturaPlayerAPI` nothing will break when we eventually remove `VideoPlayerAPI` from the codebase.

Comment (guilhermelMoraes) [reply packages/web-components/src/components/cta/video-cta-composite.ts:80]: Makes sense. Will do that, @annawen1

**Review by jeffchew — APPROVED** [review]
LGTM, thank you for jumping on this!

**Review by annawen1 — CHANGES_REQUESTED** [review]
I'm still seeing the `VideoPlayerAPI` used in other components like `CardCTA`, `LightboxMediaViewer`, etc. Can we replace all instances of VideoPlayerAPI wherever it's being used?

**Review by annawen1 — APPROVED** [review]
Approving this as another PR will be opened off master branch to remove all instances of `VideoPlayerAPI`

**Pull request merged**